docs/manual.tex

```diff
@@ -1,16 +1,34 @@
 \documentclass[a4paper]{report}
 \usepackage{microtype}
+\usepackage{imakeidx}
```

Comment:
How comes `imakeidx` instead of regular `makeidx`?

Reply:
The main known difference is that imakeidx supports multiple indices (e.g. concepts and words). The reason I used it is that imakeidx supports options, in this case to be included in the table of contents and render in three columns. There exist more variants, like index, indextools, robustindex, etc. Basically, I picked the easy path. Alternate rendering tools like hevea and plastex had limited support for other packages than makeidx and imakeidx, but that's probably a non-issue in tex4ht.